src/dodal/beamlines/i03.py

```diff
@@ -252,3 +253,17 @@ def attenuator(
         wait_for_connection,
         fake_with_ophyd_sim,
     )
+
+
+@skip_device(lambda: BL == "s03")
+def flux(wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False) -> Flux:
+    """Get the i03 synchrotron device, instantiate it if it hasn't already been.
+    If this is called when already instantiated in i03, it will return the existing object.
+    """
+    return device_instantiation(
+        Flux,
+        "flux",
+        "",
```

Comment:
Should: General preference is to put the prefix here rather than in the device, see https://github.com/DiamondLightSource/dodal/wiki/Device-Standards